pkgs/by-name/ma/marge-bot/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, python3
+, fetchFromGitLab
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "marge-bot";
+  version = "0.10.1";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "marge-org";
+    repo = "marge-bot";
+    rev = version;
+    hash = "sha256-2L7c/NEKyjscwpyf/5GtWXr7Ig14IQlRR5IbDYxp8jA=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.cfg --replace "--flake8 --pylint --cov=marge" ""
+  '';
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    configargparse
+    maya
+    pyyaml
+    requests
+  ];
+
+  nativeCheckInputs = with python3.pkgs; [ pytestCheckHook ];
+  disabledTests = [
+    # test is broken when run under Nix ("unittest.mock.InvalidSpecError: Cannot spec a Mock object.")
```

Too long line. It looks better split like this:

```suggestion
    # fails when run under Nix:
    #     unittest.mock.InvalidSpecError: Cannot spec a Mock object.
```

---

Changed the indentation (but left the text as 'broken' since it's likely a test-specific breakage rather than a genuine issue).